taca/utils/bioinfo_tab.py

```diff
@@ -148,17 +148,21 @@ def get_ss_projects(run_dir):
         logger.error('Cannot find RunParameters.xml or runParameters.xml in the run folder for run {}'.format(run_dir))
         return []
     rp = RunParametersParser(os.path.join(run_dir, run_parameters_file))
-    try:
-        runtype = rp.data['RunParameters']['Setup']['Flowcell']
-    except KeyError:
-        logger.warn('Parsing runParameters to fetch instrument type, '
-                    'not found Flowcell information in it. Using ApplicationName')
+    if 'Setup' in rp.data['RunParameters']:
         try:
-            runtype = rp.data['RunParameters']['Setup'].get('ApplicationName', '')
+            runtype = rp.data['RunParameters']['Setup']['Flowcell']
         except KeyError:
```

Comment:
Instead of catching exceptions, maybe an if with 
`runtype = rp.data['RunParameters']['Setup'].get('Flowcell', '')
 if not runtype:
   logger.warn('...')
   
`